include/screencast_common.h

```diff
@@ -123,6 +123,8 @@ struct xdpw_wlr_output {
 
 void randname(char *buf);
 int anonymous_shm_open(void);
+struct wl_buffer *import_shm_buffer(struct xdpw_screencast_instance *cast, int fd,
```

Comment:
Maybe rename to `import_wl_shm_buffer` to make this clear this imports the shm buffer into the Wayland compositor?